addons/account/models/account_payment_term.py

```diff
@@ -196,7 +196,7 @@ def _compute_terms(self, date_ref, currency, company, tax_amount, tax_amount_cur
 
         #We sort by the last line of a term
         term_lines = self.line_ids.sorted(lambda l: l == self.line_ids[-1])
-        last_percent_line = term_lines.filtered(lambda l: l.value == 'percent')[-1]
```

Comment:
This should be enough. Other changes is not needed
```py
        last_percent_line = term_lines.filtered(lambda l: l.value == 'percent')[-1:]
```

Reply:
Hello @yelizariev,
Changes have been done.
Thanks!